README.md

```diff
@@ -84,4 +93,3 @@ limitations under the License.
 [johnjohndoe-github]: https://github.com/johnjohndoe
 [pentabarf-github]: https://github.com/nevs/pentabarf
 [tuxmobil-github]: https://github.com/tuxmobil/CampFahrplan
-
```

Comment:
Please leave an empty line at the end of the file as `gitk` bugs about this somewhat.

Reply:
did not notice that visual studio code is removing the empty line.
I re-added it.